docker-compose.ci.yml

```diff
@@ -0,0 +1,14 @@
+version: "2.4"
+services:
+  terraform:
+    image: quay.io/azavea/terraform:0.12.13
```

Comment:
I'd be curious to use a more recent version of Terraform here. I created a PR to add the most recent 0.12.x patch release (0.12.28) to `docker-terraform` here: https://github.com/azavea/docker-terraform/pull/17.

Reply:
Upgrading it to `0.12.29` after your most recent changes in that PR. I'll hold off on merging until the `docker-terraform` PR is merged and the image is available, I've got it built locally for now.

Reply:
The PR has been merged and the 0.12.29 tag now exists on Quay.

A deprecated syntax warning pops up that's related to `terraform-aws-vpc`. I opened https://github.com/azavea/terraform-aws-vpc/issues/27 to address it, but it isn't fatal, so I don't think it blocks us here.